release/serve_tests/workloads/autoscaling_multi_deployment.py

```diff
@@ -170,11 +173,12 @@ def main(
         f"Deploying with min {min_replicas} and max {max_replicas}"
         f"target replicas ....\n"
     )
-    setup_multi_deployment_replicas(min_replicas, max_replicas, num_deployments)
+    all_endpoints = setup_multi_deployment_replicas(
```

Comment:
I don't think this affects behavior, but I believe `all_endpoints` is now [a list of application names](https://github.com/ray-project/ray/pull/40372/files#diff-6fea00d422393140b3646543a76fb6bdf3e3085da5cdda7561556e51873ddcedR122), not routes.

Reply:
The variable name might be a bit deceiving, but the route prefixes are set to the application names, which is why I'm just returning the list of application names directly. And since the endpoint is passed to [this function](https://github.com/ray-project/ray/blob/master/release/serve_tests/workloads/serve_test_cluster_utils.py#L82), it's not prefixed with a `"/"`.

Reply:
Makes sense, thanks for clarifying!